cardano-testnet/test/cardano-testnet-test/Cardano/Testnet/Test/Cli/Alonzo/LeadershipSchedule.hs

```diff
@@ -0,0 +1,514 @@
+{-# LANGUAGE BlockArguments #-}
+{-# LANGUAGE DisambiguateRecordFields #-}
+{-# LANGUAGE NamedFieldPuns #-}
+{-# LANGUAGE OverloadedStrings #-}
+{-# LANGUAGE ScopedTypeVariables #-}
+{-# LANGUAGE TypeApplications #-}
```

Comment:
Why was this added?

Reply:
Copied this file to write a new test and erroneously `git add`ed it :shrug:

Removed :+1: